serverlessworkflow/modules/ROOT/pages/core/defining-an-input-schema-for-workflows.adoc

```diff
@@ -1 +1,46 @@
-//= Defining an input schema for workflows
\ No newline at end of file
+= Defining an input schema for {context}
+:compat-mode!:
+// Metadata:
+:description: Defining input schema for Serverless Workflow
+:keywords: kogito, workflow, serverless, dataInputSchema
+
+The dataInputSchema in the link:{spec_doc_url}#workflow-definition-structure[Serverless Workflow specification] is one of the parameters in the Workflow definition.
+
+The dataInputSchema parameter is used to validate the workflow data input against a defined JSON Schema. It is important to provide a dataInputSchema, as it can be used to check if the provided workflow data input is a right one before any states are executed.
+
+The dataInputSchema can be defined as follows:
+[source,json]
+----
+"dataInputSchema": {
+   "schema": "URL_to_json_schema",
+   "failOnValidationErrors": false
+}
+----
+
+"schema" property is an URI, which holds the path to the JSON schema used to validate the workflow data input. The JSON schema file should be placed in the resources section of the project.
```

Comment:
This is not really true. Schema can be any URI. If a classpath URI (default if there is not schema), one place that is in the claspath is the resources directory. But if the URI is a file or an http, it wont be in the resources directory

Reply:
@fjtirado Thank you Francisco. I see your point, will update it accordingly.